internal/gps/cmd_unix.go

```diff
@@ -19,18 +19,15 @@ import (
 type cmd struct {
 	// ctx is provided by the caller; SIGINT is sent when it is cancelled.
 	ctx context.Context
-	// cancel is called when the graceful shutdown timeout expires.
-	cancel context.CancelFunc
-	Cmd    *exec.Cmd
+	Cmd *exec.Cmd
 }
 
 func commandContext(ctx context.Context, name string, arg ...string) cmd {
-	// Grab the caller's context and pass a derived one to CommandContext.
-	c := cmd{ctx: ctx}
-	ctx, cancel := context.WithCancel(ctx)
-	c.Cmd = exec.CommandContext(ctx, name, arg...)
-	c.cancel = cancel
-	return c
+	return cmd{ctx: ctx, Cmd: exec.Command(name, arg...)}
+}
+
+func (c cmd) kill() {
```

Comment:
I don't think this helper is helpful - just inline this into two places it's referenced.

Reply:
Signature doesn't match, so an annoymous wrapper function was required. No strong preference either way though

Reply:
Oh, you mean because it returns an error? Still seems better to spell it out. Now that I look at it, the `stopCancel` binding isn't needed either:

```
...
defer time.AfterFunc(time.Minute, func() {
  _ = c.Cmd.Process.Kill()
}).Stop()
```